templates/publisher/listing.html

```diff
@@ -162,18 +144,18 @@
             <select
               name="primary_category"
               {% if snap_categories.categories[0] %}
-                value="{{ snap_categories.categories[0].name }}"
+              value="{{ snap_categories.categories[0].name }}"
```

Comment:
From code readability point of view I think indent in the template code (between `{% if %}`, `{% for %}`, etc is more important than indent in rendered HTML.

Reply:
Ah thanks, I think my editor did some tomfoolery when moving things around